### PR TITLE
fix(client): replace placeholder text with intros for rdbms section

### PR DESCRIPTION
**Pull request by moT01**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Related to #42509

### CODE REVIEW

File: client/i18n/locales/english/intro.json [hunk 1]
```diff
@@ -295,11 +295,17 @@
...
+      "For these courses, you will use real developer tools and software like VS Code, PostgreSQL, and the Linux / Unix command line to complete interactive tutorials and build projects."
+    ],
     "blocks": {
       "learn-relational-databases": {
         "title": "Learn Relational Databases",
-        "intro": ["placeholder"]
+        "intro": [
+          "These courses start you off with basic Bash commands. Using the terminal, you will learn how to navigate and manipulate a file system, scripting in Bash, all the way to advanced usage and connecting to a database.",
+          "Next, you will learn how to create and work with relational databases using PostgreSQL, a database management system, and SQL, the language of these databases.",
+          "Finally, you will learn Git, the version control system, and GitHub, a code repositoy hosting service. Two essential tools for every developer."
```
Comment (Sembauke) [inline client/i18n/locales/english/intro.json:307]:
```suggestion
          "Finally, you will learn Git, the version control system, and GitHub, a code repository hosting service. Two essential tools for every developer."
```

**Review by Sembauke — DISMISSED** [review]

File: client/i18n/locales/english/intro.json [hunk 2]
```diff
@@ -295,11 +295,17 @@
...
-    "intro": ["placeholder"],
+    "intro": [
+      "For these courses, you will use real developer tools and software like VS Code, PostgreSQL, and the Linux / Unix command line to complete interactive tutorials and build projects."
+    ],
     "blocks": {
       "learn-relational-databases": {
         "title": "Learn Relational Databases",
-        "intro": ["placeholder"]
+        "intro": [
+          "These courses start you off with basic Bash commands. Using the terminal, you will learn how to navigate and manipulate a file system, scripting in Bash, all the way to advanced usage and connecting to a database.",
```
Comment (ShaunSHamilton) [inline client/i18n/locales/english/intro.json:305]:
```suggestion
          "These courses start you off with basic Bash commands. Using the terminal, you will learn everything from navigating and manipulating a file system, scripting in Bash, all the way to the advanced usage of and connection to a database.",
```

File: client/i18n/locales/english/intro.json [hunk 3]
```diff
@@ -295,11 +295,17 @@
...
+          "Next, you will learn how to create and work with relational databases using PostgreSQL, a database management system, and SQL, the language of these databases.",
+          "Finally, you will learn Git, the version control system, and GitHub, a code repository hosting service. Two essential tools for every developer."
```
Comment (ShaunSHamilton) [inline client/i18n/locales/english/intro.json:307]:
```suggestion
          "Finally, you will learn Git, the version control system, and GitHub, a code repository hosting service - two essential tools for every developer."
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
My nitpicks which you may just dismiss.

I struggled to read through the first sentence in a natural way. I included the `...everything from...`, because there is a `...to...`

**Review by ShaunSHamilton — DISMISSED** [review]
👍 LGTM 🚢

**Review by scissorsneedfoodtoo — DISMISSED** [review]
Everything LGTM :+1: :+1:

File: client/i18n/locales/english/intro.json [hunk 4]
```diff
@@ -295,11 +295,17 @@
   },
   "relational-databases": {
     "title": "Relational Databases",
-    "intro": ["placeholder"],
+    "intro": [
+      "For these courses, you will use real developer tools and software like VS Code, PostgreSQL, and the Linux / Unix command line to complete interactive tutorials and build projects."
```
Comment (ojeytonwilliams) [inline client/i18n/locales/english/intro.json:299]:
```suggestion
      "For these courses, you will use real developer tools and software including VS Code, PostgreSQL, and the Linux / Unix command line to complete interactive tutorials and build projects."
```
Nitpick.

File: client/i18n/locales/english/intro.json [hunk 5]
```diff
@@ -295,11 +295,17 @@
...
+    ],
     "blocks": {
       "learn-relational-databases": {
         "title": "Learn Relational Databases",
-        "intro": ["placeholder"]
+        "intro": [
+          "These courses start you off with basic Bash commands. Using the terminal, you will learn everything from navigating and manipulating a file system, scripting in Bash, all the way to the advanced usage of and connection to a database.",
```
Comment (ojeytonwilliams) [inline client/i18n/locales/english/intro.json:305]:
```suggestion
          "These courses start you off with basic Bash commands. Using the terminal, you will learn everything from navigating and manipulating a file system, scripting in Bash, all the way to advanced database usage.",
```
"advanced usage of and connection to" is a bit of a mouthful and I'm not sure mentioning "connection" adds much.

**Review by ojeytonwilliams — COMMENTED** [review]
@moT01 My two cents, to do with as you wish!

File: client/i18n/locales/english/intro.json [hunk 6]
```diff
@@ -295,11 +295,17 @@
...
-    "intro": ["placeholder"],
+    "intro": [
+      "For these courses, you will use real developer tools and software including VS Code, PostgreSQL, and the Linux / Unix command line to complete interactive tutorials and build projects."
+    ],
     "blocks": {
       "learn-relational-databases": {
         "title": "Learn Relational Databases",
-        "intro": ["placeholder"]
+        "intro": [
+          "These courses start you off with basic Bash commands. Using the terminal, you will learn everything from navigating and manipulating a file system, scripting in Bash, all the way to advanced database usage.",
```
Comment (Nirajn2311) [inline client/i18n/locales/english/intro.json:305]: Something about "start you off" doesn't sound right here. Maybe you can change it to:
```suggestion
          "In these courses you start off with the basic Bash commands. Using the terminal, you will learn everything from navigating and manipulating a file system, scripting in Bash, all the way to advanced database usage.",
```

or

```suggestion
          "These courses start off with basic Bash commands. Using the terminal, you will learn everything from navigating and manipulating a file system, scripting in Bash, all the way to advanced database usage.",
```

File: client/i18n/locales/english/intro.json [hunk 7]
```diff
@@ -295,11 +295,17 @@
...
+          "Next, you will learn how to create and work with relational databases using PostgreSQL, a database management system, and SQL, the language of these databases.",
+          "Finally, you will learn Git, the version control system, and GitHub, a code repository hosting service - two essential tools for every developer."
```
Comment (Nirajn2311) [inline client/i18n/locales/english/intro.json:307]:
```suggestion
          "Finally, you will learn Git, the version control system, and GitHub, a code repository hosting service - two essential tools of every developer."
```

**Review by Nirajn2311 — COMMENTED** [review]
Some things I felt that could be written better. It's up to you to accept or dismiss it.

**Review by gikf — APPROVED** [review]
Looks good :+1:

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Pull request merged**